daemon/images/image_pull.go

```diff
@@ -64,7 +64,7 @@ func (i *ImageService) PullImage(ctx context.Context, image, tag string, platfor
 		// we allow the image to have a non-matching architecture. The code
 		// below checks for this situation, and returns a warning to the client,
 		// as well as logging it to the daemon logs.
-		img, err := i.GetImage(image, imagetypes.GetImageOpts{Platform: platform})
+		img, err := i.GetImage(ctx, image, imagetypes.GetImageOpts{Platform: platform})
```

Comment:
`err` needs to be returned when that special case below does not apply.

Reply:
Done 👍